iidm/iidm-api/src/main/java/com/powsybl/iidm/network/RatioTapChanger.java

```diff
@@ -34,4 +34,8 @@
      */
     boolean hasLoadTapChangingCapabilities();
 
+    /**
+     * Set the load tap changing capabilities status.
+     */
+    void setLoadTapChangingCapabilities(boolean status);
```

Comment:
Change the return type of this method to return the current RatioTapChanger, to be consistent with whole IIDM API